File: python/00.py
#Sitio web de Python https://www.php.net/

#Comentario de una sola linea

"""
Comentario de varias lineas 
"""

'''
Comentario de varias lineas
'''

variable = "primera variable" 
'''En Python no existe la constante sino que hay una convención para que el programador sepa que
variable no mdificar'''
MY_CONSTANTE = "segunda variable"

#tipos de variables
string = "Este es un Sring (cadena de texto)"
integer = 11
floats = 11.56
booleno = True
none = None
lists = [1, 2, 3, 4, 5, 'bob', True, False]
dicionary = {
    "Hola" : "Adios",
    "undefined" : "undefined"
}

print("¡Hola, Python")

print(type(string))
print(type(integer))
print(type(floats))
print(type(booleno))
print(type(none))
print(type(lists))
print(type(dicionary))

File: php/00.php
<?php
//Sitio web PHP https://www.php.net/ 
//Comentario de una sola linea 
/*
   Comentario de varias lineas  
*/
$variable = "Esta es la primera variable";
const viable2 = "Esta es la segunda variable";

//yipos de variables
$string = "este es un estring (cadena de texto)";
$booleano = true;
$int = 11;
$float = 11.56;
$array = array(
    "Hola" => "Adios",
    "undefined" => "undefined",
);
$arraycorto = [
    "Hola" => "Adios",
    "undefined" => "undefined",
];

class objeto
{
    public $objeto;
}

$object = new objeto();

echo "¡Hola, PHP!";

echo get_debug_type($string);
echo get_debug_type($booleano);
echo get_debug_type($int);
echo get_debug_type($float);
echo get_debug_type($array);
echo get_debug_type($arraycorto);
echo get_debug_type($object);